plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/TrinoHiveCatalog.java

```diff
@@ -21,6 +21,7 @@
 import io.trino.filesystem.TrinoFileSystemFactory;
```

Comment:
nit: Consider changing the commit comment "Ignore internal TableAlreadyExistsException in Iceberg register table procedure" -> "Ignore internal TableAlreadyExistsException in register_table procedure"